### PR TITLE
[MOSIP-29595] Updated workflows as per reusable workflows templates

### PR DESCRIPTION
**Pull request by Rakshithb1**

### CODE REVIEW

File: .github/workflows/tag.ymal [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        type: string
+
+jobs:
+  tag-branch:
+    uses: mosip/kattu/.github/workflows/tag.yml@master
+    with:
+      TAG: ${{ inputs.TAG }}
+      BODY: ${{ inputs.BODY }}
+      PRE_RELEASE: ${{ inputs.PRE_RELEASE }}
+      DRAFT: ${{ inputs.DRAFT }}
```
[the remark this answers is not in the document]
Comment (syedsalman3753) [reply .github/workflows/tag.ymal:33]: https://github.com/mosip/id-authentication/pull/1101/files

[the remark this answers is not in the document]
Comment (syedsalman3753) [reply .github/workflows/tag.ymal:33]: please rename `tag.ymal` to `tag.yml`

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
release-changes.yml is missing

File: .github/workflows/tag.ymal (hunk 1 above)
[the remark this answers is not in the document]
Comment (Rakshithb1) [reply .github/workflows/tag.ymal]: updated requested changes

**Review by syedsalman3753 — APPROVED** [review]

**Review by ckm007 — APPROVED** [review]

**Pull request merged**